lib/_http_server.js

```diff
@@ -262,9 +262,8 @@ function writeHead(statusCode, reason, obj) {
     // Slow-case: when progressive API and header fields are passed.
     let k;
     if (obj) {
-      const keys = ObjectKeys(obj);
-      for (let i = 0; i < keys.length; i++) {
-        k = keys[i];
+      for (const key of ObjectKeys(obj)) {
+        k = key;
```

Comment:
I don’t think you need this variable anymore?

Reply:
The variable `k` is checked for `undefined` on line 271 😕 https://github.com/nodejs/node/blob/648088289d619bfb149fe90316ce0127083c4c99/lib/_http_server.js#L271-L273